### DISCUSSION TITLE
Video recordings with PlusServerRemoteControl.exe contain a lot of interruptions/delays.

### DISCUSSION THREAD

**Discussion by lbo2021** [PlusToolkit/PlusLib#937] [Q&A]
Hi,

I am recording two videos in parallel using PlusServerRemoteControl.exe. 
One video is 1920x1080 while the other is only 640x480. I am capturing each video at only 15 fps.
Even if the CPU is only 70% utilized and the RAM is only 25% utilized, the recorded videos are full of interruptions and delays.
I am recording in .mkv format. I do not want to use .igs.mha or .nrrd because the output files would be extremelly huge and for long recordings PlusServerRemoteControl.exe crashes when stopped if I use these kind of formats.
Can you please explain to me what the problem is?

Thank you

**Comment by Sunderlandkyl** [comment]
Can you attach your config files?

**Reply by lbo2021** [reply]
Thank you for your reply.

The configuration file is now attached to this message.
[Multi_Videos_Recording.zip](https://github.com/PlusToolkit/PlusLib/files/8562920/Multi_Videos_Recording.zip)

I created a .bat script to start and stop the recordings after 10 minutes. This is the content of it:
`
PlusServerRemoteControl.exe --command=START_ACQUISITION --device="CaptureDevice1" --output-file="RecordedVideo1.mkv"

PlusServerRemoteControl.exe --command=START_ACQUISITION --device="CaptureDevice2" --output-file="RecordedVideo2.mkv"

timeout /t 600

PlusServerRemoteControl.exe --command=STOP_ACQUISITION --device="CaptureDevice1"

PlusServerRemoteControl.exe --command=STOP_ACQUISITION --device="CaptureDevice2"

`

**Reply by Sunderlandkyl** [reply]
Follow up question, is this an issue that you observe when replaying the saved video in Plus/Slicer, or when replaying using regular media player software?

**Reply by lbo2021** [reply]
I can see the issue replaying the videos with VLC or Windows Media Player for example

**Reply by lbo2021** [reply]
@Sunderlandkyl Is there any way to enable hardware acceleration for recording with Plus Toolkit?

**Reply by Sunderlandkyl** [reply]
No, I don't think so. I tested recording in Plus and replaying in Slicer, and it seemed to work. There is probably an issue with the MKV file that is causing it to not play correctly with normal video players. I'll take a look at it.
